Starcounter.Uniform/ViewModels/UniDataTable.json.cs

```diff
@@ -137,7 +158,7 @@ public void Handle(Input.Sort action)
                     });
                 }
 
-                LoadRows?.Invoke();
+                LoadRows?.Invoke(false);
```

Comment:
don't you need to reset page to zero when changing order?

Reply:
I don't think so, order is only changing the order of the elements, not their amount. Do you think it will be better to move to page zero after order? It seemed not intuitive for me.